terraform/azure-postgresql-flexible-server/provision/main.tf

```diff
@@ -0,0 +1,102 @@
+# Copyright 2020 Pivotal Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "azurerm_resource_group" "azure-postgres" {
+  count    = length(var.resource_group) == 0 ? 1 : 0
+
+  name     = local.resource_group
+  location = var.location
+  tags     = var.labels
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
+resource "random_string" "username" {
+  length  = 16
+  special = false
+  numeric  = false
+}
+
+resource "random_password" "password" {
+  length           = 31
```

Comment:
We use `64` for the binding user password. Any reason that we can't use the same here?

Reply:
Sounds good! There is no reason to be 31 - other then the old single server offering was like that. I have changed it to be 64